### PR TITLE
Add auton routes

### PR DESCRIPTION
**Pull request by team6101**

### CODE REVIEW

File: src/main/java/frc/robot/auton/explodeDaBomb.java
Comment (nstringham) [inline src/main/java/frc/robot/auton/explodeDaBomb.java:1]: What?

Comment (Dogloverblue) [reply src/main/java/frc/robot/auton/explodeDaBomb.java:1]: I have no clue...

Comment (Alex-Dog) [reply src/main/java/frc/robot/auton/explodeDaBomb.java:1]: I wasn't there when this was written, but I'm at least glad the new people know how to make an AutonAction.

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -34,7 +35,11 @@ public class Robot extends TimedRobot {
     MotorController driveLeftChild = RobotConfigs.getLeftChild();
     MotorController driveRightParent = RobotConfigs.getRightParent();
     MotorController driveRightChild = RobotConfigs.getRightChild();
+    public static MotorController leftShooterMotor = MotorControllerFactory.create(6, MotorController.Type.SparkMax);
```
Comment (Alex-Dog) [inline src/main/java/frc/robot/Robot.java:38]: The public static part was a temporary solution, so that we could physically test the shooter.

**Review by julspr — APPROVED** [review]
looks good

**Pull request merged**